install/install.go

```diff
@@ -313,6 +321,17 @@ func (k *K8sInstaller) getCiliumVersion() semver.Version {
 	return v
 }
 
+func (k *K8sInstaller) getImagesSHA() string {
+	ersion := strings.TrimPrefix(k.params.Version, "v")
+	_, err := versioncheck.Version(ersion)
+	// If we got an error then it means this is a commit SHA that the user
+	// wants to install on all images.
+	if err != nil {
```

Comment:
Hmm.. Does this same check also pass "latest" as a "SHA" as it does not parse as a version?

Reply:
If so then we'd not need image-tag option at all?

Reply:
Let's keep the image-tag to have better semantics